momo_modules/multi_sensor_module/src/app_main.c

```diff
@@ -174,17 +195,14 @@ void acquire_pulse()
 
 void read_pulses()
 {
-	mib_buffer[0] = pulse_count() & 0xFF;
-	mib_buffer[1] = pulse_count() >> 8;
+	mib_buffer[0] = counter & 0xFF;
```

Comment:
This is fixed in the new code version.  There are separate mib calls now for reading the current state of the counters and clearing them.